packages/paste-core/utilities/grid/src/Grid.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {useTheme} from '@twilio-paste/theme';
+import {ThemeShape} from '@twilio-paste/theme-tokens';
+import {MarginProps, Space, SpaceProps} from '@twilio-paste/style-props';
+import {safelySpreadBoxProps} from '@twilio-paste/box';
+import {Flex, Vertical} from '@twilio-paste/flex';
+
+export interface GridProps extends SpaceProps {
+  children: NonNullable<React.ReactNode>;
+  gutter?: Space;
+  vertical?: Vertical;
+}
+
+const getNegativeMargin = ({gutter}: GridProps, theme: ThemeShape): {} => {
+  if (Array.isArray(gutter)) {
+    return (gutter as []).map((value: string) => {
+      return `-${theme.space[value]}`;
+    });
+  }
+
+  if (gutter) {
+    return `-${theme.space[gutter as string]}`;
+  }
+
+  return 'auto';
+};
+
+const getGridStyles = (props: GridProps): MarginProps => {
+  const theme = useTheme();
+  const marginStyles: MarginProps = {
+    marginLeft: getNegativeMargin(props, theme),
+    marginRight: getNegativeMargin(props, theme),
+  };
+
+  return marginStyles;
+};
+
+const Grid: React.FC<GridProps> = ({children, gutter, marginTop, marginBottom, vertical, ...props}) => {
+  const count = React.Children.count(children);
+
+  const GridColumns = React.useMemo(
+    () =>
+      React.Children.map(children, child =>
+        React.isValidElement(child) ? React.cloneElement(child, {count, gutter, vertical}) : child
+      ),
+    [children]
+  );
+
+  const GridStyles = React.useMemo(() => getGridStyles({children, gutter, vertical}), [{children, gutter, vertical}]);
```

Comment:
Same issue on the second argument here (object wrapping)

Reply:
Other than `gutter`, I don't think `children` and `vertical` are being used here?

Reply:
Since `children` is a required prop, if I omit from here I get TS errors.